src/App.js

```diff
@@ -46,27 +49,49 @@ const theme = createTheme({
 
 export default function App() {
   const [devMode, setDevMode] = React.useState(getIsDevMode);
+  const [userState, setUserState] = React.useState({
+    userName: "",
+    isLoggedIn: false,
+  });
+
+  const refresh = () => {
+    axios
+      .get("https://world.openfoodfacts.org/cgi/auth.pl", {
+        withCredentials: true,
+      })
+      .then((rep) => {
+        const cookieUserName = off.getUsername();
+        setUserState({ userName: cookieUserName, isLoggedIn: true });
+      })
+      .catch((err) => {
+        setUserState({ userName: "", isLoggedIn: false });
+      });
+  };
+
+  React.useEffect(refresh, []);
```

Comment:
just wondering what would be a good way of keeping track of the user loggin status... if the effect runs only on mount it will run once, if the effect runs every render if may be too much...